terraclassic/asset.json

```diff
@@ -3384,10 +3384,12 @@
   },
   {
     "id": "terra1x62dfgzrsksyemwuljhw9x0qcxefd3tpp4ued8",
+    "entity": "DoKwon HUB",
```

Comment:
Hi @reglisosa! Thanks for contributing to `yacar`, I was wondering if you could you also add the social links for "DoKwon Hub" in the `entity.json` for terraclassic, with the "name" field being "DoKwon Hub"?

Reply:
Added! please check and let me know if I should change the keys

Reply:
@reglisosa the social links should go in the `terraclassic/entity.json` file, not the current `terraclassic/asset.json`, could you help me with that change?

Reply:
Right! sorry I didn't pay close attention. Fixed now